### PR TITLE
CAD-1464 select builder when running automated RTview testing

### PR DESCRIPTION
**Pull request by CodiePP**
* select the builder when running "runTest.sh"

can be:
```
./runTest.sh --stack-nix ~/Downloads/selenium-server-standalone-3.141.59.jar
```

### CODE REVIEW

File: cardano-rt-view/test/runTest.sh [hunk 1]
```diff
@@ -17,31 +25,35 @@ echo "Remove old UNIX-socket..."
 rm -f "${UNIX_SOCKET}"
 
 echo "Launch cardano-rt-view-service..."
-nohup stack exec -- "${RT_VIEW_EXE}" --config "${RT_VIEW_CONFIG}" \
-                                     --static "${RT_VIEW_STATIC_DIR}" \
-                                     --port "${RT_VIEW_WEB_PORT}" &
```
Comment (denisshevchenko) [inline cardano-rt-view/test/runTest.sh:22]: Why did you remove `nohup`? It's truly daemonizes the `cardano-rt-view-service` process, I thought it's better solution.

**Review by denisshevchenko — APPROVED** [review]
LGTM, except one question, about `nohup`.

File: cardano-rt-view/test/runTest.sh (hunk 1 above)
Comment (CodiePP) [reply cardano-rt-view/test/runTest.sh:22]: can't use it with the "run" script function.
